src/Components/Compute/AuctionBlock/Auction.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+
+import AuctionBlock from '../../UI/AuctionBlock/AuctionBlock';
+
+export default class Auction extends React.Component {
+  constructor(props) {
+    super(props);
+    this.initialState = {
+      biddingSequence: props.biddingSequence,
+      highestBid: null,
+      winningPlayer: null,
+      currentBid: null,
+    };
+    this.state = { ...this.initialState };
```

Comment:
IMO, You don't need additional `initialState` property in the constructior. directly use `this.state`